docs/java/features/extensions/extension-library/curconv/currency-conversion.md

```diff
@@ -13,69 +13,225 @@ keywords:
 - currency conversion
 - extension
 ---
+import useBaseUrl from '@docusaurus/useBaseUrl'
+import MvnBadge from './MvnBadge'
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<MvnBadge />
 
 ## Overview
 
-With its expressive name **Currency conversion** extension is a convenient library that targets a ubiquitous business case of converting between currencies in your cloud application.
+Currency Conversion is an extension library built on SAP Cloud SDK. You can use this library to facilitate currency exchange rate conversions.
+
+:::note
+The library does not provide any market data out-of-the-box. You must provide the currency exchange rates to the library from your own data provider sources.
+:::
+
+You can use the library for the following functions:
+
+- Direct one-to-one conversions, from a source currency to a target currency.
+- Indirect conversions by using inverse rates or a "reference currency".
+- Fixed rate conversions, for when you have the exact currency exchange rate that you want to use in your conversion operations.
+- Non-fixed rate conversions, for when you have a set of exchange rates, as opposed to the exact rate. The library will pick the "best rate" from the set depending on [various factors](#non-fixed-rate).
+
+## Getting Started
 
-## Getting started
+Adding the Currency Conversion library to your list of dependencies will allow you to perform the following tasks:
 
-Edit you `pom.xml` file and add Currency Conversion library to your dependencies.
+- Convert a single or list of amounts from one currency to another by using a fixed rate.
+- Convert a single or list of amounts from one currency to another by using exchange rates provided at runtime.
+- Convert a single or list of amounts from one currency to another by using tenant-specific rates provided at runtime.
+- Configure the conversion process for users or tenants accessing your application.
+
+Add the dependency in your pom.xml file under the existing *dependencies* tag as follows:
 
 ```xml
 <dependency>
    <groupId>com.sap.cloud.sdk.services</groupId>
    <artifactId>currency-conversion-core</artifactId>
-   <!-- Please, use the latest version of the dependency -->
+   <!-- Please use the latest version of the dependency -->
    <version>3.X.X</version>
 </dependency>
 ```
-If your IDE haven't resolved the dependencies run `mvn clean install` in the root directory of you project.
+The library works within the tenant context, therefore, depending on the landscape you operate in, add one of the following dependencies in your pom.xml file, in addition to that of the library's. Adding this additional dependency helps the library detect your tenants automatically. Without it, you would have to manually pass this information. Therefore, we recommend adding one or more of the following dependencies, based on your landscape.
 
-## Features
+If you want to run the library in the Cloud Foundry environment, use the following:
```

Comment:
Use tabs to provide content for different alternatives instead of showing both contents at the same time with an "if": https://v2.docusaurus.io/docs/markdown-features/#tabs

Reply:
We liked the Tabs idea and considered it but when we tried to include the xml code snippet, we ran into errors.
![Tabs Error](https://user-images.githubusercontent.com/37209163/92923234-be93aa00-f454-11ea-9361-34062e5914d9.png)
If there's a fix for this, we'd be happy to implement.